### ISSUE TITLE
Menu Bar Missing OS X

### ISSUE THREAD

**Issue by chrispadgett** [ProjectIRoniC/OpenPointMesh#64]
On OS X the Menu Bar that allows the user to select application settings is missing.

![os x superbuild](https://cloud.githubusercontent.com/assets/3378627/15125796/f336ad8a-15eb-11e6-8369-28a6cc791177.png)

**Comment by paulwrodriguez** [comment]
I don't believe this issue will be fixed with the css changes that will fix most of the other inconsistencies we have between os's. Not sure why this is happeneing since we set  `QCoreApplication::setAttribute(Qt::AA_DontUseNativeMenuBar);` Linux type systems sometimes puts the menu bar at the top of the screen as opposed to on the GUI itself.

**Comment by chrispadgett** [comment]
When running the application the menu bar items are not in the top menu bar or the GUI itself.
